README.md

```diff
@@ -3,6 +3,9 @@
     A secure, fast, and general-purpose OS kernel written in Rust and compatible with Linux
 </p>
 
+[![Check License](https://github.com/asterinas/asterinas/actions/workflows/license_check.yml/badge.svg)](https://github.com/asterinas/asterinas/actions/workflows/license_check.yml)
```

Comment:
I suggest replacing the badge for check license with that for OSDK test. The former is not that important to have a badge.

Another suggestion is to adjust the position of the badges. The current version renders as below.

<img width="1233" alt="image" src="https://github.com/asterinas/asterinas/assets/568208/e767bc79-a1be-4c3e-b750-dfbe690a9c9e">

I gave it some tries and ended up with the following version. I think it looks better.

<img width="1203" alt="image" src="https://github.com/asterinas/asterinas/assets/568208/e0c538f2-4423-42f1-b05a-78bd34ce5544">

The code is below.

```html
<p align="center">
    <img src="docs/src/images/logo_en.svg" alt="asterinas-logo" width="620"><br>
    A secure, fast, and general-purpose OS kernel written in Rust and compatible with Linux<br/>
    <a href="https://github.com/asterinas/asterinas/actions/workflows/osdk_test.yml"><img src="https://github.com/asterinas/asterinas/actions/workflows/osdk_test.yml/badge.svg" alt="OSDK Test" style="max-width: 100%;"></a>
    <a href="https://github.com/asterinas/asterinas/actions/workflows/kernel_test.yml"><img src="https://github.com/asterinas/asterinas/actions/workflows/kernel_test.yml/badge.svg" alt="Kernel Test" style="max-width: 100%;"></a>
    <br/>
</p>

English | [中文版](README_CN.md)
```